jablib/src/main/java/org/jabref/logic/importer/fileformat/books/EpubImporter.java

```diff
@@ -0,0 +1,39 @@
+package org.jabref.logic.importer.fileformat.books;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+import org.jabref.logic.importer.TikaImporter;
+import org.jabref.logic.importer.util.Constants;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.util.FileType;
+import org.jabref.logic.util.StandardFileType;
+
+public class EpubImporter extends TikaImporter {
```

Comment:
Some refernce to the format would be nice.

(Even if it self-explanatory here; but mayb e some more non-obvious onformatoin can be found there)